compiler/circle-execution-plan/src/ExecutionPlanner.cpp

```diff
@@ -303,33 +303,59 @@ void ExecutionPlanner::create_alloc_node_inform_vector(bool null_consts, bool nu
     }
 
     // Scratchpad If needed
-    uint32_t scratchpad_size = 0;
-    switch (circle_node->opcode())
+    std::vector<uint32_t> scratchpad_sizes;
```

Comment:
change to vector due to some operations (now it is SVDF and according to #8335 BatchMul) can have more than one scratchpad tensors.